packages/components/src/dot/src/Dot.css

```diff
@@ -1,26 +1,29 @@
 .o-ui-dot {
```

Comment:
![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/d1e87107-986d-422f-adab-2e9c0d7e37f0)
This seems very pale vs the old design
![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/08b35a9f-163f-427e-8132-0f4e4404544f)

Reply:
hm ya I'm not sure why.. I didn't do that part but I'll check it out

Reply:
This look disabled at the current state